app/ssttpcalculator/CalculatorService.scala

```diff
@@ -122,10 +122,15 @@ class CalculatorService @Inject() (
       debit <- overallDebits.map(_.filterNot(_.dueDate.isAfter(calculatorInput.startDate)))
     } yield calculateHistoricInterest(debit)).sum
 
-    // Calculate interest for the first 7 days until the initial payment is actually taken out of the taxpayer's account
-    val initialPaymentInterest: BigDecimal = if (calculatorInput.initialPayment > 0)
-      calculateInitialPaymentInterest(calculatorInput.debits.filter(_.dueDate.isBefore(calculatorInput.startDate.plusWeeks(1))))
```

Comment:
This was a reference to the initial 7 days via `.plusWeeks(1)`.

I'm worried there's other uses of `.plusWeeks(1)` in this class but I don't understand them enough to know if they should be replaced with 10 days. 
e.g. on lines:
- 182/187
- 240/245
- 400/405